File: problemas/1000/1048.dart
import 'dart:io';

void main() {
  final salario = double.parse(stdin.readLineSync());

  final indice = salario <= 400.00
    ? 0.15
    : salario <= 800.00
      ? 0.12
      : salario <= 1200.00
        ? 0.1
        : salario <= 2000.00
          ? 0.07
          : 0.04;

  final reajuste = salario * indice;

  stdout.writeln('Novo salario: ${(salario + reajuste).toStringAsFixed(2)}');
  stdout.writeln('Reajuste ganho: ${reajuste.toStringAsFixed(2)}');
  stdout.writeln('Em percentual: ${(indice * 100).toStringAsFixed(0)} %');
}

File: problemas/1000/1048.py
salario = float(input())

indice = 0.15 if salario <= 400.00 else\
    0.12 if salario <= 800.00 else\
    0.1 if salario <= 1200.00 else\
    0.07 if salario <= 2000.00 else\
    0.04

reajuste = salario * indice

print(f'Novo salario: {salario + reajuste:.2f}')
print(f'Reajuste ganho: {reajuste:.2f}')
print(f'Em percentual: {indice * 100:.0f} %')
